src/types/onyx/OnfidoWallet.ts

```diff
@@ -0,0 +1,21 @@
+type OnfidoWallet = {
+    /** Unique identifier returned from openOnfidoFlow then re-sent to ActivateWallet with Onfido response data */
+    applicantID?: string;
+
+    /** Token used to initialize the Onfido SDK token */
+    sdkToken?: string;
+
+    /** Loading state to provide feedback when we are waiting for a request to finish */
+    loading?: boolean;
+
+    /** Error message to inform the user of any problem that might occur */
+    error?: string;
```

Comment:
This should be errors

```suggestion
    errors?: Record<string, string>;
```

Reply:
Thank you, adjusted!